README.md

```diff
@@ -12,16 +12,16 @@ stakeholders can be confident about the direction the project is evolving in.
 
 ## How to submit an RFC
 
-1. Copy `template.md` into `text/YYYY-MM-DD-my-feature.md`.
+1. Copy `template.md` into `text/PRID-my-feature.md`.
 2. Write the document and fill in the blanks.
 3. Submit a pull request.
 
 ## Timeline of an RFC
 
 1. An RFC is submitted as a PR.
 2. Discussion takes place, and the text is revised in response.
-3. The PR is merged or closed when at least two project maintainers reach
-   consensus.
+3. The PR is accepted or rejected when at least two project maintainers reach consensus.
+4. If accepted, create a tracking issue, refer it in the RFC, and finalize by merging.
```

Comment:
We may need bot to create it automatically.

Reply:
FYI, in the rust community, rfc is merged once it is accepted. The tracking issue can be created later.

Reply:
It is a bit tricky because we don't have a central repository and either a tracking issue under

- tikv/tikv
- tikv/pd
- tikv/raft-rs

I don't see a good automatic way and think it can be a later discussion. The problem is we want a way to tracking how a feature gets implemented.

Reply:
> FYI, in the rust community, rfc is merged once it is accepted. The tracking issue can be created later.

@andylokandy Yes it works. I write as is because we want a way to tracking how a feature gets implemented. And after merging the RFC, creating a tracking issue, and ask the contributor to update the RFC think a bit awkward.

Reply:
When the RFC is ready to merge, the person merging it can create a tracking issue in the right repo, add the link to the PR, then merge it.

Reply:
> When the RFC is ready to merge, the person merging it can create a tracking issue in the right repo, add the link to the PR, then merge it.

Yes. That is exactly what in my mind.